examples/step-37/step-37.cc

```diff
@@ -814,7 +814,8 @@ namespace Step37
         (update_gradients | update_JxW_values | update_quadrature_points);
       std::shared_ptr<MatrixFree<dim, double>> system_mf_storage(
         new MatrixFree<dim, double>());
-      system_mf_storage->reinit(dof_handler,
+      system_mf_storage->reinit(StaticMappingQ1<dim>::mapping,
```

Comment:
I think it would be more instructive to add a `MappingQ1` member to the class here and pass it to `interpolate_boundary_values` and other functions that implicitly assume a mapping as well.